h/views/admin/groups.py

```diff
@@ -46,7 +46,7 @@ def groups_index(context, request):
     renderer="h:templates/admin/groups_create.html.jinja2",
     permission="admin_groups",
 )
-class GroupCreateController(object):
+class GroupCreateView(object):
```

Comment:
The class should be called `GroupCreateViews` (plural) because it's a class that groups multiple views (the methods) into one class. See <https://docs.pylonsproject.org/projects/pyramid/en/latest/quick_tutorial/view_classes.html>.